docs/source/sysadmin_guide/configure_sgx.rst

```diff
@@ -139,6 +139,7 @@ Install Sawtooth
     $ sudo apt-get update
     $ sudo apt-get install -y -q \
```

Comment:
Let's remove -q so that this command is consistent with the non-SGX install command.